src/libraries/System.Private.CoreLib/src/System/DateTime.Unix.cs

```diff
@@ -15,10 +15,10 @@ public static DateTime UtcNow
             }
         }
 
-        private static DateTime FromFileTimeLeapSecondsAware(ulong fileTime) => default;
-        private static ulong ToFileTimeLeapSecondsAware(long ticks) => default;
+        private static DateTime FromFileTimeLeapSecondsAware(ulong _) => default;
```

Comment:
Similar here. We have been trying to keep the signatures across the different partial types identical.

Reply:
I'd personally prefer `_` because the methods are never called but moved to pragma too

Reply:
There are still number of other places left in this PR that mix actual argument name and `_` across different partial type definitions. We should treat all of them same way.

Reply:
I think I caught all now

Reply:
The three methods in `ComActivator` appear to still have this problem:

https://github.com/dotnet/runtime/pull/63015/files#diff-cea663fe3ea038ade6cee5ec9b485483a02cb4e7816cb6992699976e538568a7R15

Reply:
> The three methods in ComActivator appear to still have this problem:

They are not in partial type

Reply:
Isn't the intent the same though? It is the same class that must have the same method signatures. But instead of C# calling the methods, the native runtime is calling them.

Reply:
I find this illogical so don't really know what was the original intention. I can change this to have parameter names with pragma which says they are not needed in this implementation if you think that will make the code more readable.

Reply:
> so don't really know what was the original intention

I meant the original intention of the rule from @jkotas [above](https://github.com/dotnet/runtime/pull/63015#discussion_r775095535):

> I do not think it is an improvement to omit the argument name in cases like this one where the method signature has to be symmetric with other method signatures.

In `ComActivator`, the method signatures need to be symmetric, even though it isn't a `partial` class.

Reply:
Replaced with pragma